iothub_service_client/tests/iothub_srv_client_auth_ut/iothub_service_client_auth_ut.cpp

```diff
@@ -57,6 +57,7 @@ static STRING_HANDLE TEST_VALUE_STRING_HANDLE = (STRING_HANDLE)0x4646;
 
 static const char* TEST_CHAR_PTR = "TestString";
 static const char* TEST_CONNECTION_STRING = "HostName=aaa.bbb.net;SharedAccessKeyName=xxx;SharedAccessKey=yyy";
+static const char* TEST_SAS_CONNECTION_STRING = "HostName=aaa.bbb.net;SharedAccessSignature=yyy";
```

Comment:
I am looking at the iothub_service_client\samples\iothub_registrymanager_sample\iothub_registrymanager_sample.c and seeing that we can enhance the sample here to a) manually validate this and b) add a comment there to demonstrate we can use DPS and the new API to make this work. Please do so.

Reply:
Please skip my comment (b) since I realized the DPS setup is way too much for this sample. A smoke test would be nice on this new API for now. 

---
In reply to: [246548962](https://github.com/Azure/azure-iot-sdk-c/pull/800#discussion_r246548962) [](ancestors = 246548962)

Reply:
we tested this with our DM agent … the agent uses DPS to get and store device connection info in our device TPM, the agent then gets a device SharedAccessSignature from the TPM and uses that to create new modules.